Frameworks/Foundation/NSDecimalNumber.mm

```diff
@@ -428,9 +565,86 @@ + (NSDecimalNumber*)maximumDecimalNumber {
  @Status Stub
  @Notes
 */
-+ (NSDecimalNumber*)minimumDecimalNumber {
+- (instancetype)initWithFloat:(float)num {
     UNIMPLEMENTED();
     return StubReturn();
 }
 
+/**
+ @Status Interoperable
+*/
+- (instancetype)initWithInt:(int)num {
+    return [self initWithMantissa:((num < 0) ? -num : num) exponent:0 isNegative:((num < 0) ? YES : NO)];
```

Comment:
>-num [](start = 47, length = 4)

If I do initWithInt:INT_MIN, this will overflow and produce undefined behavior. Depending on how the compiler feels, it might do what you want and it might not.